src/frontend/src/app/footer/footer.component.css

```diff
@@ -1,2 +1,5 @@
-
+.blockquote-footer{
+  position: static;
+  bottom: 0;
+}
 
```

Comment:
moze zostawic tego starego footera? poki co dziala, a potem mozna sie pobawic w to, zeby on byl przyczepiony do dolu

Reply:
dla mnie ok